pkg/cmd/server/kubernetes/master.go

```diff
@@ -128,6 +129,11 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) ([]string, error
 	return messages, nil
 }
 
+func newMasterLeases(storage storage.Interface) election.Leases {
+	leaseTTL := uint64((master.DefaultEndpointReconcilerInterval + 5*time.Second) / time.Second) // add 5 seconds for wiggle room
```

Comment:
I'd drop a comment in here about *why* you're dividing by `time.Second`, so people don't get confused in the future.